validation/shallow_water_model/near_global_shallow_water_quarter_degree.jl

```diff
@@ -182,7 +181,7 @@ model = ShallowWaterModel(grid = grid,
 ##### Initial condition:
 #####
 
-h_init = deepcopy(1e1 .+ maximum(bat) .- bat) 
+h_init = - deepcopy(bat) .+ 10 # we should add a positive value to avoid a vanishing layer
```

Comment:
Do we need to initialize `h` with `-bathymetry`? (btw the variable names here could be improved...)

Reply:
Good question and observation.  

I defined `h` to be the total depth, which seems to be the correct thing to do.

I am happy to change the names after we get something working.

Reply:
I meant really that `bat` is that mammal that flies or the stick we play baseball.